spec/enumerables_spec.rb

```diff
@@ -5,6 +5,7 @@
   describe '#my_each' do
     it 'Return a valid enumerable' do
       expect(testArray.my_each).not_to be_a(String)
+      expect(testArray.my_each).to be_a(Enumerable)
     end
```

Comment:
Let's not add two `expect` in one example. Separate it into two different examples as they are testing different cases.

> You should have one `it` block for each expected behaviour. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/itblock.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))